libs/wingsdk/src/cloud/counter.ts

```diff
@@ -89,6 +89,13 @@ export interface ICounterClient {
    * @inflight
    */
   peek(): Promise<number>;
+
+  /**
+   * Reset a counter to a given value.
+   * @param value value to reset (default is 0)
+   * @inflight
+   */
+  reset(value?: number): Promise<void>;
```

Comment:
This value is not optional in the sdk spec https://github.com/winglang/wing/pull/1633/files, but given this is a reset method and not set I think it makes for a nicer DX. I think we should update the spec to reflect this default behavior @Chriscbr wdyt?

Reply:
Yea makes sense. I think that as a developer I would expect `counter.reset()` to just set to counter to 0.
I'll change the spec to with your decision as I fix the nits.

Reply:
Of course 0 should be the default. 

Make sure to document with a @default doc tag

Reply:
curious - why 0 over the initial value as default?

Reply:
You are right. That's a good point.

But... this requires "capturing" the value from preflight to the inflight client. In wing this is trivial but in typescript it require a few hoola hoops.

Maybe open an issue to follow up.

Reply:
Resolving for now